model/GlobalConfig.php

```diff
@@ -1,243 +1,243 @@
-<?php
```

Comment:
Please don't make style fixes in unrelated code as part of a feature PR. It's practically impossible to see whether you've changed anything in this file, making it difficult to review. I suggest you revert this part and make it a separate PR if you like.